scripts/Microsoft.Xaml.Behaviors.Uwp.Managed.nuspec

```diff
@@ -35,18 +35,6 @@
         <file target="lib\net8.0-windows10.0.19041.0\Microsoft.Xaml.Interactivity.pdb"  src="..\out\uwp\BehaviorsSDKManaged\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.pdb" />
         <file target="lib\net8.0-windows10.0.19041.0\Microsoft.Xaml.Interactivity.xml"  src="..\out\uwp\BehaviorsSDKManaged\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.xml" />
         <file target="lib\net8.0-windows10.0.19041.0\Microsoft.Xaml.Interactivity.pri"  src="..\out\uwp\BehaviorsSDKManaged\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.pri" />
-
-        <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactivity.Design.dll"       src="..\out\BehaviorsSDKManaged\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.Design.dll" />
-        <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactivity.Design.pdb"       src="..\out\BehaviorsSDKManaged\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.Design.pdb" />
-
-        <file target="lib\net8.0-windows10.0.19041.0\Design\Microsoft.Xaml.Interactivity.Design.dll"       src="..\out\BehaviorsSDKManaged\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.Design.dll" />
-        <file target="lib\net8.0-windows10.0.19041.0\Design\Microsoft.Xaml.Interactivity.Design.pdb"       src="..\out\BehaviorsSDKManaged\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.Design.pdb" />
-
-        <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactivity.DesignTools.dll"   src="..\out\BehaviorsSDKManaged\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.DesignTools.dll" />
-        <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactivity.DesignTools.pdb"   src="..\out\BehaviorsSDKManaged\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.DesignTools.pdb" />
-
```

Comment:
We want to keep *.DesignTools.dll's - just not *.Design.dll

Reply:
Updated! 🙂